ArduCopter/mode.cpp

```diff
@@ -768,7 +768,7 @@ void Mode::land_run_horizontal_control()
         // interpolate for 1m above that
         const float attitude_limit_cd = linear_interpolate(700, copter.aparm.angle_max, get_alt_above_ground_cm(),
                                                      g2.wp_navalt_min*100U, (g2.wp_navalt_min+1)*100U);
-        const float thrust_vector_max = sinf(radians(attitude_limit_cd * 0.01f)) * GRAVITY_MSS * 100.0f;
+        const float thrust_vector_max = sinf(attitude_limit_cd * radians(0.01f)) * GRAVITY_MSS * 100.0f;
```

Comment:
I would strongly prefer a cd_to_rad() function or similar

Reply:
IMHO cdeg_to_rad would be more readable.